examples/color/color_cycler.py

```diff
@@ -24,15 +24,16 @@
 
 # 1. Setting prop cycle on default rc parameter
 plt.rc('lines', linewidth=4)
-plt.rc('axes', prop_cycle=(cycler('color', ['r', 'g', 'b', 'y']) +
-                           cycler('linestyle', ['-', '--', ':', '-.'])))
+plt.rc('axes', prop_cycle=(cycler(color=['r', 'g', 'b', 'y']) +
+                           cycler(linestyle=['-', '--', ':', '-.'])))
 fig, (ax0, ax1) = plt.subplots(nrows=2)
 ax0.plot(yy)
 ax0.set_title('Set default color cycle to rgby')
 
 # 2. Define prop cycle for single set of axes
-ax1.set_prop_cycle(cycler('color', ['c', 'm', 'y', 'k']) +
-                   cycler('lw', [1, 2, 3, 4]))
+#    While you can provide a cycler to `.set_prop_cycle`, it is easier to
+#    directly pass the properties as keywords.
```

Comment:
This wording is correct, but I think it might be misleading. They kwarg form for `ax.set_prop_cycle()` is for convenience. The assumption is that most of the time, users would be adding cyclers together (zip() form) rather than multiplying them (product() form). So, we made the former easier to do. Don't know if that changes your thinking on how the wording here should be.

Reply:
Rewritten to be a bit more explicit.

In general, I think the example could benefit from some work. But that would be part of a separate PR. So I don't want to invest too much time into polishing the words here.